changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 0.6.3
+
+- CLI package: Modify WebpackError to handle undefined `stats` value.
+- Header package: Relabel "Account Settings" as "Settings and Accounts" to be more consistent with mobile app terminology; redirect old account settings page to new Manage My Account page.
```

Comment:
This is an internal detail, the only thing that changed was fixtures and tests so not necessary for consumers to know about